$OpenBSD: patch-src_unexelf_c,v 1.1 2010/09/09 19:14:23 jasper Exp $

Fix build on OpenBSD/alpha, which is an ELF platform.

--- src/unexelf.c.orig	Thu Sep  9 19:27:28 2010
+++ src/unexelf.c	Thu Sep  9 19:28:00 2010
@@ -1079,7 +1079,7 @@ unexec (new_name, old_name, data_start, bss_start, ent
       memcpy (NEW_SECTION_H (nn).sh_offset + new_base, src,
 	      NEW_SECTION_H (nn).sh_size);
 
-#ifdef __alpha__
+#if defined(__alpha__) && !defined(__OpenBSD__)
       /* Update Alpha COFF symbol table: */
       if (strcmp (old_section_names + OLD_SECTION_H (n).sh_name, ".mdebug")
 	  == 0)
